sites/walmart.py

```diff
@@ -45,7 +45,7 @@ def monitor(self):
             "accept-language": "en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7",
             "cache-control": "max-age=0",
             "upgrade-insecure-requests": "1",
-            "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.193 Safari/537.36"
+            "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36"
```

Comment:
Can we move user-agent to just be a single variable? Ideally I think this is something we should define in settings I think so it can be universal across both best buy and walmart. I can move this elsewhere if you would prefer later. Ideally we shouldnt have to change in many places though, just 1 variable.

Reply:
I agree. One global variable and maybe change it based on Mac/Windows? Not sure if that matters

Reply:
Agreed I didn't want to touch too much. I'll wrap it up into a single place for future changes.